packages/web/src/algorithms/types.ts

```diff
@@ -62,8 +62,10 @@ export interface AminoacidSubstitution {
   gene: string
 }
 
-export interface SubstitutionsWithAminoacids extends NucleotideSubstitution {
-  aaSubstitutions: AminoacidSubstitution[]
+export interface AminoacidDeletion {
+  refAA: Aminoacid
+  codon: number
+  gene: string
 }
```

Comment:
This is the new data structure for the AA deletions:

For the end user this can be visualized as `${gene}:${refAA}${codon}-` (e.g. `ORF10:D123-`).